docs/pages/access-controls/guides/access-requests-slack.mdx

```diff
@@ -0,0 +1,347 @@
+---
+title: Teleport integration with Slack
+description: This guide explains how to set up a Slack plugin for Teleport for privilege elevation approvals.
+h1: Teleport Slack Plugin Setup
+---
+
+This guide examines how to configure Teleport for use with Slack.
+
+Slack integration allows you to treat Teleport access and permission requests within Slack messages as well as through the command line.
```

Comment:
We can make it less self referential if we zoom in on users' benefit, for example:

Slack integration allows you to review, grant and deny access to SSH servers using command line or Web UI.